docs/03 - Implementing the Anchor Server/Communication/Platform API.yml

```diff
@@ -290,18 +290,29 @@ components:
           enum: [31]
         kind:
           type: string
-          enum: ["receive"]
+          enum: ["receive", "withdraw", "deposit"]
         status:
           type: string
           enum: [
-            "pending_sender",
-            "pending_stellar",
-            "pending_customer_info_update",
-            "pending_receiver",
-            "pending_external",
-            "completed",
-            "error"
-          ]
+            pending_anchor,
```

Comment:
I think it's a good idea to group together SEP-24, SEP-31 and common statuses with a comment
```
# SEP-24
<statuses>
# SEP-31
<statuses> 
```

Reply:
Changed.